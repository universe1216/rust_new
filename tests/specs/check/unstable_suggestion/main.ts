/// <reference no-default-lib="true" />
/// <reference lib="deno.ns" />

// unstable apis removed here, so should error
Deno.dlopen("path/to/lib", {});
