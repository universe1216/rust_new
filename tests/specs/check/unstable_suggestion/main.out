Check file:///[WILDLINE]main.ts
error: TS2551 [ERROR]: Property 'dlopen' does not exist on type 'typeof Deno'. Did you mean 'open'? 'Deno.dlopen' is an unstable API. Did you mean 'open'? If not, try changing the 'lib' compiler option to include 'deno.unstable' or add a triple-slash directive to the top of your entrypoint (main file): /// <reference lib="deno.unstable" />
Deno.dlopen("path/to/lib", {});
     ~~~~~~
    at file:///[WILDLINE]/main.ts:5:6

    'open' is declared here.
      export function open(
                      ~~~~
        at asset:///lib.deno.ns.d.ts:[WILDLINE]
